### PR TITLE
Lf 3051 fix estimated annual revenue calculation in update estimated crop revenue

### PR DESCRIPTION
**Pull request by Duncan-Brain**
**Description**

Now that units is fixed the calculation no longer needs to manually convert units to calculate. This relies upon the databaseUnits of the input values to be the same. I have not made this explicit but could consider it in the future -- 

`if (dbunitA=dbUnitB) {
calculate
} else {
convert then calculate
}`

Jira link: [LF-3051](https://lite-farm.atlassian.net/browse/LF-3051)

**Type of change**

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**How Has This Been Tested?**

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Passes test case
- [x] UI components visually reviewed on desktop view
- [ ] UI components visually reviewed on mobile view
- [x] Other (please explain): Manual commenting out hidden value and testing several situations

**Checklist:**

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] The precommit and linting ran successfully
- [ ] I have added or updated language tags for text that's part of the UI
- [ ] I have added "MISSING" for all new language tags to languages I don't speak
- [ ] I have added [the GNU General Public License](https://lite-farm.atlassian.net/l/cp/BT0Dd7WW) to all new files

[LF-3051]: https://lite-farm.atlassian.net/browse/LF-3051?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/webapp/src/components/Finances/UpdateEstimatedCropRevenue/index.jsx [hunk 1]
```diff
@@ -57,30 +55,15 @@ function PureUpdateEstimatedCropRevenue({ system, managementPlan, onGoBack, onSu
...
-      setValue(ESTIMATED_ANNUAL_REVENUE, revenue);
-    } else {
-      const adjustedAnnualYield = roundToTwoDecimal(
-        convert(convertedAnnualYield).from(annualYieldUnit.value).to(pricePerMassUnit.value),
-      );
-      const revenue = roundToTwoDecimal(convertedPricePerMass * adjustedAnnualYield);
-      setValue(ESTIMATED_ANNUAL_REVENUE, revenue);
+  // Calculate Revenue
+  useEffect(() => {
+    if (pricePerMass && annualYield) {
```
Comment (SayakaOno) [inline packages/webapp/src/components/Finances/UpdateEstimatedCropRevenue/index.jsx:63]: When pricePerMass or annualYield is "0", Estimated annual revenue is not calculated. Was that intentional?
I think it's okay, but just confirming!

**Review by SayakaOno — COMMENTED** [review]
So clean!!! 😆

File: packages/webapp/src/components/Finances/UpdateEstimatedCropRevenue/index.jsx (hunk 1 above)
Comment (Duncan-Brain) [reply packages/webapp/src/components/Finances/UpdateEstimatedCropRevenue/index.jsx:63]: @SayakaOno I think it was intentional haha -- why estimate zero revenue?

**Review by kathyavini — COMMENTED** [review]
Wow, the changes to Unit made THAT way simpler :rofl: Very nice.

Could you be so kind as to fix the CSS now though ❤️ Still not lining up!

**Comment by Duncan-Brain** [comment]
@kathyavini OKay... but its on you if the happy path doesnt pass 😂 

Went with the 13.5 px to keep the css matching between as `align-self` doesnt work on Input

**Comment by kathyavini** [comment]
@Duncan-Brain `align-self: center` would go on the `.currency` class (it definitely does work!) but the result is visually the same and so my perfectionism has been is mollified, thank you 🙏 

Happy path will pass! ... eventually... 😬

**Review by kathyavini — APPROVED** [review]

File: packages/webapp/src/components/Finances/UpdateEstimatedCropRevenue/index.jsx (hunk 1 above)
Comment (SayakaOno) [reply packages/webapp/src/components/Finances/UpdateEstimatedCropRevenue/index.jsx:63]: Okay good!

**Review by SayakaOno — APPROVED** [review]

**Pull request merged**